pkg/deb/Makefile.wasm

```diff
@@ -6,14 +6,19 @@ MODULE_SUMMARY_wasm=	WASM module for NGINX Unit
 MODULE_VERSION_wasm=	$(VERSION)
 MODULE_RELEASE_wasm=	1
 
-MODULE_CONFARGS_wasm=	wasm --include-path=\$$(CURDIR)/pkg/contrib/wasmtime/crates/c-api/include --lib-path=\$$(CURDIR)/pkg/contrib/wasmtime/target/release \&\& ./configure wasm-wasi-component
-MODULE_MAKEARGS_wasm=	wasm wasm-wasi-component CFLAGS=\"\$$(shell grep ^CFLAGS \$$(BUILDDIR_\$$*)/build/Makefile | cut -d' ' -f 3-) -Wno-missing-prototypes\"
-MODULE_INSTARGS_wasm=	wasm-install wasm-wasi-component-install
+MODULE_CONFARGS_wasm=	wasm-wasi-component
+MODULE_MAKEARGS_wasm=	wasm-wasi-component CFLAGS=\"\$$(shell grep ^CFLAGS \$$(BUILDDIR_\$$*)/build/Makefile | cut -d' ' -f 3-) -Wno-missing-prototypes\"
+MODULE_INSTARGS_wasm=	wasm-wasi-component-install
+
+ifeq (,$(findstring $(CODENAME),bullseye focal))
+MODULE_CONFARGS_wasm+=	\&\& ./configure wasm --include-path=\$$(CURDIR)/pkg/contrib/wasmtime/artifacts/include --lib-path=\$$(CURDIR)/pkg/contrib/wasmtime/artifacts/lib
+MODULE_MAKEARGS_wasm+=	wasm
+MODULE_INSTARGS_wasm+=	wasm-install
 
 MODULE_SOURCES_wasm=
 
-BUILD_DEPENDS_wasm=
-MODULE_BUILD_DEPENDS_wasm=
+BUILD_DEPENDS_wasm=	cmake
+MODULE_BUILD_DEPENDS_wasm=,cmake
```

Comment:
Comma or space?

Reply:
MODULE_BUILD_DEPENDS_* is appended to a string of comma-separated tokens over at https://github.com/nginx/unit/blob/2ac4a75/pkg/deb/debian.module/control.in#L8 to properly write a control file.

and BUILD_DEPENDS_* is appended to a space-separated list to feed to the package manager directly, so needs to have a space or a tab.